eng/packages/http-client-csharp/generator/Azure.Generator/src/Azure.Generator.csproj

```diff
@@ -3,6 +3,7 @@
     <TargetFramework>net8.0</TargetFramework>
     <PackageId>Azure.Generator</PackageId>
     <Version>1.0.0-beta.1</Version>
+		<Nullable>enable</Nullable>
```

Comment:
consider to put this in https://github.com/Azure/azure-sdk-for-net/blob/main/eng/packages/http-client-csharp/generator/Directory.Build.props, so that this will apply to all future projects. 
How about the test projects?

Reply:
I think we should put it in a global place. Test projects aslo need this.